docs/webui.md

```diff
@@ -0,0 +1,222 @@
+---
+title: 'Gravitino web UI'
+slug: /webui
+keyword: webui
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+license: 'Copyright 2023 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.'
+---
+import Image from '@theme/IdealImage'
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+This document primarily outlines how developers can manage metadata within Gravitino using web UI, the graphical interface accessible through a web browser as an alternative to manually writing code.
+
+Currently, you can integrate [OAuth settings](security.md) to view, add, modify, and delete metalakes, create catalogs, and view catalogs, schemas, and tables, among other functions.
+
+:::caution
+More features are under development. For detailed capabilities, please refer to the [Manage metadata using Gravitino](manage-metadata-using-gravitino.md) document.
+:::
+
+Complete the installation, and open the browser [http://localhost:8090](http://localhost:8090).
```

Comment:
Link to document on how to "Completer the installation". It is probably better to say "Build and deploy the Gravitino Web UI and open it in a browser at ..."